catalog/update.go

```diff
@@ -29,8 +29,22 @@ func updateFunctionFactory(pathPrefix string, updatable []string) func(context.C
 			}, field) {
 				continue
 			}
-			old, new := d.GetChange(field)
-			if !d.HasChange(field) {
+
+			var old, new interface{}
+			if field == "columns" {
+				old, new = d.GetChange("column")
+				if !d.HasChange("column") {
+					continue
+				}
+			} else {
+				old, new = d.GetChange(field)
+				if !d.HasChange(field) {
+					continue
+				}
+			}
+
+			//ignore storage_location for MANAGED tables
+			if field == "storage_location" && d.Get("table_type") != "EXTERNAL" {
```

Comment:
shouldn't it be handled as suppress diff?
Also, should we handle the case when people will change from managed to external?

Reply:
`table_type` cannot be updated